images/php/fpm/entrypoints/51-production-detection.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [[ -z "${PHP_ERROR_REPORTING}" ]]; then
+    if [[ ${LAGOON_ENVIRONMENT_TYPE} == "production" ]]; then
+        export PHP_ERROR_REPORTING="E_ALL & ~E_DEPRECATED & ~E_STRICT & ~E_NOTICE"
+    else
```

Comment:
I wonder if we need this else, as we already have defaults in `00-lagoon-php.ini.tpl` that are the same as in the else I think this is not really necessary?

Reply:
Agreed. Resolved in commit 9ea8185e92817094bef86cd67eb2ac2664d7b407